HOOKS.md

```diff
@@ -327,11 +329,30 @@ useRenderActivity(
   }) => React.Element
 ): ({
   activity: Activity,
-  timestampClassName: string
+  nextVisibleActivity: Activity
 }) => React.Element
 ```
 
-This function is for rendering an activity and its attachments inside a React element. Because of the parent-child relationship, the caller will need to pass a render function in order for the attachment to create a render function for the activity. When rendering the activity, the caller will need to pass `activity` and `timestampClassName`. This function is a composition of `activityRendererMiddleware`, which is passed as a prop.
+This function is for rendering an activity and its attachments inside a React element. Because of the parent-child relationship, the caller will need to pass a render function in order for the attachment to create a render function for the activity. When rendering the activity, the caller will need to pass `activity` and `nextVisibleActivity`. This function is a composition of `activityRendererMiddleware`, which is passed as a prop.
+
+Because not all activities are rendered, for example, event activities are not rendered. The `nextVisibleActivity` is the pointer to next visible activity and is intended for activity status renderer on grouping timestamps for adjacent activities.
```

Comment:
```suggestion
Note that not all activities are rendered, e.g. the event activity. Because of this, those activities will not be rendered. The `nextVisibleActivity` is the pointer to the next visible activity and is intended for the activity status renderer on grouping timestamps for adjacent activities.
```
Add link to ACTIVITIES.md